.gitlab-ci.yml

```diff
@@ -5,34 +5,44 @@ stages:
   - build
   - test
 
-# disabled because cannot find -luv -lcap when building
+# disabled because gcc9 can't build net_if.hpp
```

Comment:
Is this referring to the xenial bug with conflicting net/if.h and linux/if.h includes?  That should be fixed now.

Reply:
related to those changes, gcc9 gives this:
```
In file included from /builds/neuroscr/loki-network/llarp/ev/ev.h:15,
                  from /builds/neuroscr/loki-network/llarp/ev/ev.hpp:5,
                  from /builds/neuroscr/loki-network/llarp/util/thread/logic.hpp:4,
                  from /builds/neuroscr/loki-network/llarp/util/thread/logic.cpp:1:
 /builds/neuroscr/loki-network/llarp/net/net_if.hpp:8:38: error: expected initializer before '__THROW'
     8 | if_nametoindex(const char* __ifname) __THROW;
       |                                      ^~~~~~~
 make[3]: *** [llarp/CMakeFiles/lokinet-util.dir/build.make:666: llarp/CMakeFiles/lokinet-util.dir/util/thread/logic.cpp.o] Error 1
```

Reply:
This is probably not gcc 9 but rather alpine's libc.

Reply:
Well is it forward looking? Do we care about alpine? I can see use caring about making sure our code base works with newer gccs but not sure about how libc works irt this